### DISCUSSION TITLE
n error occurred while refreshing accessories: Error: System serial not found. at SS3Platform.discoverSimpliSafeDevices

### DISCUSSION THREAD

**Discussion by zulumonkeymetallic** [homebridge-simplisafe3/homebridge-simplisafe3#342] [Q&A]
Hey team., 

I was wondering if anyone else has come across anything like the errors below when running in a docker container on pi. 

![image](https://user-images.githubusercontent.com/94960648/169395341-ba1018e2-f492-47a0-8ab9-13dbea3e6483.png)

Child process ended [5/19/2022, 8:59:47 PM] [Home Alarm] An error occurred while refreshing accessories: Error: System serial not found. at SS3Platform.discoverSimpliSafeDevices
![Screenshot 2022-05-19 at 21 01 53](https://user-images.githubusercontent.com/94960648/169395421-cab89cb3-b7c5-4918-9755-f8ff48dbf7ff.png)

**Comment by shamoon** [comment]
Your other plugin is failing too. Perhaps theres something wrong in your setup with reaching the internet?

Alternatively, are you specifying the serial number in your config?

**Comment by vasco747** [comment]
Did you find out what the problem was? I have the same error
